README.md

```diff
@@ -103,6 +103,19 @@ resource.sort(:description, :created_at).filter(:active => true).each do |item|
 end
 ```
 
+### Returning select fields
+Resources have a `only` method that accepts any number of field options to return only the selected fields. Note that each call to `only` adds to any previous fields.
```

Comment:
"field options" -> "field names"

Also, see my comment about the adding to previous fields behavior. If we make that change we'll need to update the doc, too.

Reply:
Renamed "field options" to "field keys" as it does not actually accept field names either